products/BellHybrid/apps/application-bell-powernap/presenter/PowerNapProgressPresenter.cpp

```diff
@@ -56,6 +57,14 @@ namespace app::powernap
     }
     void PowerNapProgressPresenter::onNapFinished()
     {
+        const std::string frontlightEnabled =
+            settings->getValue(bell::settings::Alarm::lightActive, settings::SettingsScope::Global);
+
+        if (frontlightEnabled == std::string("1")) {
```

Comment:
There is an `AlarmLightOnOffModel` you can use to obtain that value. It probably should be moved to the `common` too.

Reply:
Added AlarmLightOnOffModel to PowerNapProgressPresenter